client/components/action-card/test/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @jest-environment jsdom
+ */
+import '@testing-library/jest-dom';
+import { shallow } from 'enzyme';
+import { axe, toHaveNoViolations } from 'jest-axe';
+expect.extend( toHaveNoViolations );
+import ActionCard from '..';
+
+const props = {
+	headerText: 'This is a header text',
+	mainText:
+		'This is a description of the action. It gives a bit more detail and explains what we are inviting the user to do.',
+	buttonText: 'Call to action!',
+	buttonIcon: 'external',
+	buttonPrimary: true,
+	buttonHref: 'https://wordpress.com',
+	buttonTarget: '_blank',
+	buttonOnClick: null,
+};
+
+describe( 'ActionCard basic tests', () => {
+	test( 'should not blow up and have proper CSS class', () => {
```

Comment:
Oooh, yeah. I need to break my brain out of the gherkin world it's been in. Instead, this could read something like `test('Render proper CSS class)' 

I think the action-card should have a few more checks. I'd remove this test when filling those out. Which I would definitely want to do since this seems like a fairly important component. If possible, I'd want to start adding tests to components in priority of how often the components are used with the assumption that those analytics are available.

That 'should not blow up' step appears a few times but could be cleaned up pretty easily. I'm more concerned about the components that don't have a test folder at all...